main/ms.f90

```diff
@@ -443,7 +444,9 @@ Program main
     end if
     if(comm_is_root(2)) then
       ix_m=NX_table(NXY_s)
```

Comment:
@yhirokawa san, thank you for your suggestion! Actually, the line was useless. So, I deleted it.